cla_common/laalaa.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+
+class LaaLaaError(Exception):
+    pass
+
+
+class LaalaaProviderCategoriesApiClient:
```

Comment:
I think it's usual to be explicit about inheritance. Is this the same as inheriting from `object`?